packages/fxa-settings/src/pages/ResetPassword/ResetPasswordWithRecoveryKeyVerified/container.tsx

```diff
@@ -62,10 +59,13 @@ const ResetPasswordWithRecoveryKeyVerifiedContainer = ({
       // This is fine - we navigate to the confirmed page and carry on.
       navigateWithQuery('/reset_password_verified', { replace: true });
     }
-  }, [recoveryKey, navigateWithQuery]);
+    if (!email) {
```

Comment:
Should we swap and put this email check above recovery check? If there's no email there is likely no recovery key either and this might cause an extra stop via `/reset_password_verified` before getting redirected again to signin?

Reply:
Good idea, fixed!